### ISSUE TITLE
Quit not working when close to tray is enabled

### ISSUE THREAD

**Issue by Bogo36** [amanharwara/altus#295]
Quit not working when close to tray is enabled.

**Comment by coredevz** [comment]
Yes... When the close to tray then the quit button is also not working

**Comment by Bogo36** [comment]
Still not working with 5.3.0

**Comment by Daniel-Kalus** [comment]
Still not working in 5.4.0 and still very annoying. Basically no way to quit the app other than task manager when close to tray is enabled. Should be able to close it via right click on tray icon -> Quit, or in the app by choosing File -> Quit.

**Comment by Bogo36** [comment]
Still not working on 5.5.1 even its stated as fixed, but it is not.

**Comment by amanharwara** [comment]
Hi, thanks for letting me know. Can you try [v5.5.2](https://github.com/amanharwara/altus/releases/tag/5.5.2)?
